scm/driver/azure/repo.go

```diff
@@ -35,7 +56,32 @@ func (s *RepositoryService) FindCombinedStatus(ctx context.Context, repo, ref st
 }
 
 func (s *RepositoryService) Create(ctx context.Context, input *scm.RepositoryInput) (*scm.Repository, *scm.Response, error) {
-	return nil, nil, scm.ErrNotSupported
+	ro, err := decodeRepo(input.Namespace + "/" + input.Name)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	proj, res, err := s.getProject(ctx, ro)
+	if err != nil {
+		return nil, res, err
+	}
+
+	in := repository{
+		Name: input.Name,
+		Project: struct {
+			ID    string `json:"id"`
+			Name  string `json:"name"`
+			State string `json:"state"`
+			URL   string `json:"url"`
+		}{
+			ID: proj.ID,
+		},
+	}
+	out := new(repository)
+	endpoint := fmt.Sprintf("https://dev.azure.com/%s/%s/_apis/git/repositories?api-version=6.0", ro.org, ro.project)
```

Comment:
should this be hardcoded to dev.azure.com?

Reply:
thanks for the catch